graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -406,8 +406,8 @@ int64_t PickByEtype(
     PickedType* picked_data_ptr);
 
 template <
-    bool NonUniform, bool Replace, typename ProbsType = float,
-    typename PickedType>
+    bool NonUniform, bool Replace, typename ProbsType, typename PickedType,
```

Comment:
C++ requires template parameters with default values to come after the other template parameters. So, it is necessary to delete the default from ProbsType. I don't know how it was compiling before.